### PR TITLE
[all] bump Kubernetes version to v1.4.3

### PR DESCRIPTION
**Pull request by colhom**
\cc @cgag @sym3tri 

Running conformance test now.

### CODE REVIEW

**Comment by aaronlevy** [comment]
Looks like picked up some extraneous changes to kube-dashboard addon and also some vendor code.

File: multi-node/aws/vendor/golang.org/x/crypto/openpgp/armor/armor_test.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ func TestDecodeEncode(t *testing.T) {
 		t.Errorf("len(result.Header): got:%d want:1", len(result.Header))
 	}
 	v, ok := result.Header["Version"]
-	if !ok || v != "GnuPG v1.4.10 (GNU/Linux)" {
+	if !ok || v != "GnuPG v1.4.30 (GNU/Linux)" {
 		t.Errorf("result.Header: got:%#v", result.Header)
 	}
```
Comment (cgag) [inline multi-node/aws/vendor/golang.org/x/crypto/openpgp/armor/armor_test.go]: I assume all these GnuPG changes just got caught in the crossfire?

**Comment by cgag** [comment]
Yeah, the dashboard looks like it's still at 1.4.1. 

`docker: Tag v1.4.3 not found in repository gcr.io/google_containers/kubernetes-dashboard-amd64.`

**Comment by aaronlevy** [comment]
lgtm

**Pull request merged**